Eastern-Research-Group/mywaterway
# Feature/multi state links

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3430625

## Main Changes:
* On the state page, changed the link at the bottom into an accordion with multiple links. This comes from the attains/metrics service.
* On the state page, changed the metrics and intro text to come from  the attains/metrics service.
* Removed the introText lookup file. 
* Updated cypress tests to work with these changes.

## Steps To Test:
1. Open the chrome dev tools
2. Navigate to http://localhost:3000/state/PA/water-quality-overview
3. Verify the metrics are visible and formatted similarly to the dev site. 
  * Note: The service does not return the metrics in the same order as the file, so the order probably won't match.
4. Verify the state intro text is visible.
5. Expand the "More information for Pennsylvania" accordion and verify 2 links are visible.
6. Block the attains/metrics service
7. Verify the error messages are visible
8. Navigate to http://localhost:3000/state/FL/water-quality-overview
9. Verify the percentage metrics are formatted correctly
10. Run the state.spec.js cypress tests and verify they work.
